ixdiagnose/artifacts/items/directory.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from distutils.dir_util import copy_tree
```

Comment:
distutils has been deprecated and will be removed in 3.12.
```suggestion
from shutil import copytree
```